zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -232,6 +232,11 @@ public Note getNote() {
   }
 
   public boolean isEnabled() {
+    //avoid NullPointException when method isTerminated
+    //invoked in ParagraphListenerImpl.afterStatusChange
+    if (config == null) {
+      return true;
```

Comment:
if no config, should this return false?

Reply:
@felixcheung  According  to code：
```java
      Boolean enabled = (Boolean) config.get("enabled");
      return enabled == null || enabled.booleanValue();
 ```
we can know that if no enabled configured, return true, so I  think the default value of enabled is true. So I returned true at this place.